lib/src/polyline.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math' as math;
+
+import 'package:turf/helpers.dart';
+
+class Polyline {
```

Comment:
what about a small description here?

Reply:
Resolved in https://github.com/dartclub/turf_dart/pull/96